examples/mobile/UIComponents/components/content/textinput.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+	var page = document.getElementById("textinput-demo"),
+		textInputs = document.querySelectorAll(".input-hint");
+
+	page.addEventListener("pagebeforeshow", function () {
+		var idx,
+			onFocusBound,
+			onBlurBound;
+
+		onFocusBound = onFocus.bind(null);
+		onBlurBound = onBlur.bind(null);
```

Comment:
How about assigning the value with defining the value?

Reply:
I simplified code now